### ISSUE TITLE
CSV file does not include columns for wallet address or txid.

### ISSUE THREAD

**Issue by dan-da** [bitpay/copay#3071]
For accounting purposes I need a list of all wallet addresses that have been used in transactions.  I was expecting the CSV file to include these, but it does not.  So it seems that the only way I can get this info through the UI is to manually click on each transaction then "See it on the blockchain", figure out which is my wallet address and copy/paste into a file.  This is beyond tedious.

fyi:  the primary reason I need this info is to be able to generate a report with USD price (cost basis) at the time of each transaction to provide an accountant.  ie, it is important for business/legal reasons.  Since Copay does not provide USD amounts, I must generate them myself.  I wrote a tool for this purpose:  http://github.com/dan-da/bitprices.

Also, the txid is present but is appended to the Note field, eg:  
    user memo Txid: 433464354....

Really the txid should be in its own column.

So to summarize, there should be two additional columns:  "Wallet Address" and "Transaction ID".

Also, in looking at this in more detail I notice that there are no rows for change.  What I wish to see in the CSV file is a complete reporting of all fund flows.   That means all inputs and outputs that involve an address in my wallet.

I recognize this is a more comprehensive view.  Perhaps there could be an option for "Detailed Report" when clicking the download link.

If there is some alternate way to obtain the full list of wallet addresses ever used (including change addresses) please provide instructions.

**Comment by matiu** [comment]
because we are compatible with LibraTax , it is not posible to add new columns to the export, that is why TXID and NOTES where included in the same column.

Adding the fiat equivalence at the tx time is an open ticket: https://github.com/bitpay/copay/issues/2961

thanks.

**Comment by dan-da** [comment]
It surely is possible.  There could easily be a checkbox for "Detailed Report" that provides more info.  As I suggested above.

Providing addresses associated with each transaction is basic functionality.
